specification/examples/CreateQueue.example.xml

```diff
@@ -0,0 +1,67 @@
+PUT https://daschulttest1.servicebus.windows.net/testQueuePath?api-version=2017-04 HTTP/1.1
+Authorization: SharedAccessSignature sr=https%3A%2F%2Fdaschulttest1.servicebus.windows.net%2FtestQueuePath%2F&sig=bCEWzr1DOgXB5XCeNjWJU2zfhH1%2F%2Fca56gVNovbPPUY%3D&se=1539812750&skn=RootManageSharedAccessKey
```

Comment:
Is this the real SharedAccessKey? If so please delete this key from the portal, remove it from here and make sure to either not have this header or fill it with dummy value.